### ISSUE TITLE
No ability to print multiple logs

### ISSUE THREAD

**Issue by jsardev** [getgrit/gritql#377]
I'm a bit confused about why running `log` logs only the first log and then nothing else. Could all the `log` executions be visible in `stdout`? 🤔

**Comment by morgante** [comment]
I'm not sure what you mean by this. Please provide the commands and code you use to reproduce.

**Comment by jsardev** [comment]
```
engine marzano(0.1)
language js

string_fragment() as $fragment where {
    log(message = $fragment)
}
```

```
console.log("Hello, Dan")
console.log("Hello, James")
```

https://app.grit.io/studio?key=4-_6OE96nQtRDugIa2Hep

@morgante I was expecting to see logs for both occurences, but when I run this via `grit apply` - I only see the first `log` execution.

**Comment by morgante** [comment]
Ah ok, the studio only shows the first log currently.

They should all show up on the CLI.

**Comment by jsardev** [comment]
Yeah I provided a studio link for faster showcase of the code, but I'm testing it via CLI and it doesn't work there 🤔
<img width="320" alt="CleanShot 2024-06-27 at 21 37 18@2x" src="https://github.com/getgrit/gritql/assets/11553624/b4ec722d-2eb7-4583-9a37-828a502278c8">

**Comment by morgante** [comment]
Ah it's possibly related to this: https://github.com/getgrit/gritql/blob/main/crates/marzano_messenger/src/emit.rs#L141
